receiver/k8seventsreceiver/go.mod

```diff
@@ -5,41 +5,46 @@ go 1.17
 require (
 	github.com/open-telemetry/opentelemetry-collector-contrib/internal/k8sconfig v0.41.0
 	github.com/stretchr/testify v1.7.0
-	go.opentelemetry.io/collector v0.41.1-0.20211210184707-4dcb3388a168
+	go.opentelemetry.io/collector v0.39.0
```

Comment:
This is still wrong. It should be aligned with the go.mod at the root of this repository.

Reply:
I have aligned it with the go.mod at the root of this repository.